src/style.css

```diff
@@ -84,8 +83,8 @@ a {
 
 .search-option-container,
 .filter-catogery-container {
-  height: 20rem;
-  overflow-y: scroll;
+  /* height: 20rem;
+  overflow-y: scroll; */
```

Comment:
Remove if not required. Don't submit commented code like this unless you have a very good reason to

Reply:
Noted.